docs/snippets/_sdk-reduce-latency.mdx

```diff
@@ -1,7 +1,5 @@
-By default, UID2 SDKs make API calls to a UID2 production environment server in the USA. Depending on where your users are based, you might consider choosing a server closer to your users to reduce latency.
+By default, the UID2 SDKs and Prebid.js implementation make API calls to a UID2 production environment server in the USA. Depending on where your users are based, you might consider choosing a server closer to your users to reduce latency.
```

Comment:
I think only the client-side SDKs (JS, prebid, android, ios) have a default URL. The server-side SDKs make you choose a URL (although this might change in future).
Would be better if there's a way to avoid the first sentence.

Reply:
Hi @jon8787: because this is a shared piece, it goes in both SDK and Prebid docs, so I included SDKs and Prebid in the copy. I appreciate it isn't all SDKs. Rather than getting even more specific, in the shared section I'm moving this content, which is implementation-specific, out of the shared piece to the individual documents.